code/HabitTracker/app/src/main/java/com/example/habittracker/activities/fragments/AddEventFragment.java

```diff
@@ -283,6 +283,7 @@ public void onClick(DialogInterface dialogInterface, int i) {
                             String tempLocation = location1.getText().toString();
                             String tempComments = editText1.getText().toString();
                             HabitEvent tempHabitEvent = new HabitEvent();
+                            tempHabitEvent.setHabitInDB(tempHabit,2);
```

Comment:
Same thing here

Reply:
As we discussed I'll move it slightly for clarity